### PR TITLE
Add test for Button Component and Container: close #8 and close #9

### LINKED ISSUE #8: Tests for ButtonComponent

### LINKED ISSUE #9: Tests for ButtonContainer

### PR DESCRIPTION
**Pull request by nan2iz**

### CODE REVIEW

File: src/Framework/Core/Form/Control/Button/__tests__/Button.component-test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ButtonComponent from '../Button.component';
+import i18n from '../Button.i18n';
+
+const messages = i18n.en;
+
+describe('components <ButtonComponent />', () => {
+  it('renders lebel button correctly -- string type', () => {
```
Comment (psychobolt) [inline src/Framework/Core/Form/Control/Button/__tests__/Button.component-test.js]: misspelled 'label'

File: src/Framework/Core/Form/Control/Button/__tests__/Button.container-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ButtonContainer from '../Button.container';
+
+describe('container <ButtonContainer />', () => {
+  it('container rander correctly', () => {
```
Comment (psychobolt) [inline src/Framework/Core/Form/Control/Button/__tests__/Button.container-test.js]: misspelled 'render'

File: src/Framework/Core/Form/Control/Button/__tests__/Button.component-test.js (hunk 1 above)
Comment (nan2iz) [reply src/Framework/Core/Form/Control/Button/__tests__/Button.component-test.js]: Fixed

File: src/Framework/Core/Form/Control/Button/__tests__/Button.container-test.js (hunk 2 above)
Comment (nan2iz) [reply src/Framework/Core/Form/Control/Button/__tests__/Button.container-test.js]: Fixed

**Review by psychobolt — APPROVED** [review]

**Review by nan2iz — COMMENTED** [review]
Fixed misspelling. Thanks.

**Pull request merged**